schemas.json

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
I don't think we want the schemas.json to be tracked in this repository, as this PR is pulling it from the "single source of truth" wich is the amo-blocklist-ui repository.